src/reconcile/rec_row.c

```diff
@@ -74,12 +74,12 @@ __rec_cell_build_int_key(WT_SESSION_IMPL *session, WTI_RECONCILE *r, const void
 }
 
 /*
- * __rec_cell_build_leaf_key --
+ * __wti_rec_cell_build_leaf_key --
  *     Process a key and return a WT_CELL structure and byte string to be stored on a row-store leaf
  *     page.
  */
-static int
-__rec_cell_build_leaf_key(
+int
+__wti_rec_cell_build_leaf_key(
```

Comment:
Can we ensure not building any overflow key values for delta? They cannot delta with that.

Reply:
That's good catch, I've created a ticket [here](https://jira.mongodb.org/browse/WT-15187).